### DISCUSSION TITLE
Follow Friends in the `daily.dev` Platfrom

### DISCUSSION THREAD

**Discussion by kunalkeshan** [dailydotdev/daily#839] [Feature Requests]
Connect with friends on the daily.dev platform and stay updated on their latest activity. 

Follow my friends to see what they are reading and sharing, and discover new content that aligns with your interests.

**Comment by idoshamun** [comment]
Hi @kunalkeshan,

Thanks for the suggestion. We're actually working on groups rather than followership.
You will be able to share content with the group and invite your friends.
I hope this will satisfy your needs 😬

**Reply by kunalkeshan** [reply]
That's really cool,@idoshamun! Sure, a way to connect with friends over peers also sounds like a good idea on the platform.
